libc/src/__support/fixed_point/fx_rep.h

```diff
@@ -245,6 +245,7 @@ template <> struct FXRep<long accum> {
   LIBC_INLINE_VAR static constexpr int FRACTION_LEN = LACCUM_FBIT;
   LIBC_INLINE_VAR static constexpr int TOTAL_LEN =
       SIGN_LEN + INTEGRAL_LEN + FRACTION_LEN;
+  LIBC_INLINE_VAR static constexpr int VALUE_LEN = INTEGRAL_LEN + FRACTION_LEN;
```

Comment:
also remove this in favor of `get_value_len()`?

Reply:
Yes, I must've missed this one. Thanks.

Resolved by [04db3f7](https://github.com/llvm/llvm-project/pull/114318/commits/04db3f7b880ebc904d0829e36b652f19886db6d3)